src/test/java/com/emc/object/s3/S3JerseyClientTest.java

```diff
@@ -2174,6 +2174,33 @@ public void testSetObjectAclRequestCanned() {
         //TODO - need to verify the returned acl is comparable to the canned acl
     }
 
+    @Test
+    public void testExtendObjectRetentionPeriod() throws Exception {
```

Comment:
More cases need to be added
https://confluence.cec.lab.emc.com/display/ECS/ECS+3.6+-+Retention+Enhancement+Test+Plan

Reply:
Would you give an example which test cases need to be added? This test case is only for triggering retentionUpdate via new API client.extendRetentionPeriod(). The functionality of retentionUpdate is not considered in scope.

Reply:
That's fine. I was hoping at least we could add -1 & -2 in our test case. But we should confirm with the component owner for the -2 usage.